docs/framework/20categories/20architecture/bridge-aggregation-protocols.md

```diff
@@ -1,4 +1,38 @@
+Bridge aggregation is synonymous with liquidity aggregation – meaning bridge aggregators source liquidity from token bridges, DEXs, and DEX aggregators in order to algorithmically and efficiently move funds between blockchains. 
+
+For example, if a user wants to swap USDC on Arbitrum to ETH on Ethereum, they would not be able to do so via a bridge aggregator that doesn’t support DEXs. However, a bridge aggregator with DEXs can facilitate such a cross-chain swap by bridging USDC from Arbitrum to Ethereum and then swapping it for ETH via a DEX (like Uniswap). 
+
+In essence, bridge aggregators help users find the most optimal route for a cross-chain swap from token X on blockchain A to token Y on blockchain B in a single user interface. Furthermore, they enable users to compare alternate routes based on factors like cost, speed, security, and slippage, and choose the best one that fits their requirements and preferences. 
+
+Most aggregators have an on-chain and an off-chain component for routing cross-chain transactions:
+
+* **Off-chain routing algorithm** finds the most efficient route for a cross-chain transfer. This is accomplished by comparing the routes and quotes provided by different liquidity sources (bridges, DEXs, DEX aggregators) for a certain trade and selecting a recommended route based on the rules set by the bridge aggregator’s internal routing algorithm and/or the user’s preferences. From there, an API communicates this information to the bridge aggregator’s front-end, which showcases the routes to the user. Although centralized, the off-chain component is vital for any bridge aggregator because all bridges provide their quotes and routes off-chain, thus requiring bridge aggregators, by default, to operate their routing algorithm and APIs off-chain. Moreover, doing all the computation off-chain makes delivering the most efficient route for a cross-chain transfer faster and cheaper, improving the user experience.
+
+* **On-chain smart contract aggregates** the smart contracts for each bridge, DEX, and DEX aggregator supported by the bridge aggregator and executes the cross-chain transaction after the user selects a route. However, while the bridge aggregator’s smart contract abstracts the complexity of dealing with multiple bridges and DEXs, it adds another layer of smart contract risks.
+
+Bridge aggregation protocols can be classified based on how they interact with their users:
+
+**Indirectly via other dApps**
+
+In such systems, bridge aggregation protocols work in the background and do not directly interact with the end users. Instead, the bridge aggregation protocols are integrated into a dApp’s cross-chain service offering. For example, in [MetaMask Bridges](https://portfolio.metamask.io/bridge), cross-chain transfers are executed via two bridge aggregation protocols: [LI.FI](https://li.fi/) and [Socket](https://socket.tech/).
+
+The benefits of using a bridge aggregator over an individual bridge for dApps include:
```

Comment:
I think it might be good to try and balance this list of positives with some considerations, if possible. Examples:
- aggregators enable multi-step or multi-hop bridging, which might increase execution risk 
- aggregators combine many different protocols (i.e., different bridges and DEXs) with different security properties and risk profiles in the same option set. Users have to partly trust aggregators to offer a curated set of options that minimize their risk.

Reply:
Agreed - also having their own smart contracts in some cases can be a risk vector. I'll add these considerations.

Reply:
@ArjunChand13 I updated this section in PR #44, which has been merged. Can you kindly have a look at the [updated version here](https://crosschainriskframework.github.io/framework/20categories/20architecture/architecture/#bridge-aggregation-protocols), and let me know what you think? Thanks

Reply:
Looks good to me. We can add a point separately for the smart contract risk added by aggregators.

Also, the formatting of considerations in the updated version is different from the other sections. Could you please check?

Reply:
Smart contract risk would generally fall in our "protocol implementation risk" category, but it might be worth highlighting here anyway. 

> ..the formatting of considerations in...

Good catch. I will fix it.

Reply:
Addressed in PR #60